libcontainer/configs/cgroup_unix.go

```diff
@@ -20,7 +20,7 @@ type Cgroup struct {
 	ScopePrefix string `json:"scope_prefix"`
 
 	// Resources contains various cgroups settings to apply
-	Resources *Resources `json:"resources"`
+	Resources
```

Comment:
I sort of don't understand why we need to embed it? Why just don't use value instead of pointer?